src/containers/Dashboard/Courses/Courses.js

```diff
@@ -1,33 +1,65 @@
 import React, { Component, Fragment } from "react";
 import Card from "./Card";
+import { addNotification } from "../../../utilities";
+import API from "../../../api/index";
 import "./Courses.scss";
 
 class Courses extends Component {
   constructor(props) {
     super(props);
 
     this.state = {
-      courses: ["OOP", "Algo", "DB"]
+      courses: [],
+      cards : ["Top Courses", "Most Viewed", "Top Rated"],
+      course : []
+    };
+  }
+
+  componentWillMount() {
+    API.getCourses(result => {
```

Comment:
@HarisShahidHameed move API calling in a separate function. Your component's hook should be clean and obvious.

Reply:
Moved to a separate function.